bessctl/conf/samples/arp.bess

```diff
@@ -0,0 +1,11 @@
+import scapy.all as scapy
+
+eth_header = scapy.Ether(src='02:1e:67:9f:4d:ae', dst='ff:ff:ff:ff:ff:ff')
+arp_header = scapy.ARP(op=1, pdst='1.1.1.1')
+pkt = eth_header/arp_header
+packets = [str(pkt)]
+
+arp = ArpResponder(name="arp")
```

Comment:
`name="arp"` is set but not used below. For conciseness, this line can be replaced with either:
```
arp::ArpResponder()  # this is identical to the original statement
```
or
```
arp = ArpResponder()  # the resulting (default) name will be arp_responder0
```